docs/content/any/project/changelogs/2021-09-29.md

```diff
@@ -66,24 +69,25 @@ const user = {name: "notanadmin"};
 await oso.isAllowed(user, "read", "document");
 ```
 
-Before this change, this `isAllowed()` call would silently fail. Oso would look up the
-nonexistent `isAdmin` property on the `{name: "notanadmin"}` object, which returns
-`undefined`. Since `undefined` is not equivalent to `true`, the `isAllowed()` call would
-fail, but the developer wouldn't have any idea that they'd written a policy that referenced
-a nonexistent property.
+Before this change, this `isAllowed()` call would silently fail. Oso would look
+up the nonexistent `isAdmin` property on the `{name: "notanadmin"}` object,
+which returns `undefined`. Since `undefined` is not equivalent to `true`, the
+`isAllowed()` call would fail, but the developer wouldn't have any idea that
+they'd written a policy that referenced a nonexistent property.
 
 Now, this will error since `isAdmin` is not a property on the `user` object.
 
-This will still work if (1) the `isAdmin` property is explicitly set to `undefined`
-(e.g., `user = {isAdmin: undefined}`) or (2) the property check is moved into the head
-of the rule:
+This will still work if (1) the `isAdmin` property is explicitly set to
+`undefined` (e.g., `user = {isAdmin: undefined}`) or (2) the property check is
+moved into the head of the rule:
 
 ```polar
 allow(user: {isAdmin: true}, "read", "document");
 ```
-since specializers will not error if a property doesn't exist (but the `{name: "notanadmin"}`
-object will fail to match the specializer and the `isAllowed()` call will still correctly
-return `false`).
+
+since specializers will not error if a property doesn't exist (but the `{name:
+"notanadmin"}` object will fail to match the specializer and the `isAllowed()`
+call will still correctly return `false`).
 
 ##### Optional `types` parameter to `registerClass()` renamed to `fields`
```

Comment:
We also made this change in the Python library, we need to call that out too. Also maybe move this one up to the top of the list, since it's relatively short & otherwise folks might miss it

Reply:
Good catch — gonna do this as part of a follow-up PR that adds a couple small changelog entries.